### PR TITLE
JSON to YAML

### PR DESCRIPTION
**Pull request by jalovisko**
Provides a script to convert JSON files to YAML configurations suitable to be used by PyCAM.

The script is located in `pycam/json2yml.py` (run `--help`).

There is also an example JSON file that can be used for testing `pycam/params.json`.

The JSON converted YAML can then be imported to the CLI version of PyCAM (`pycam/run_cli.py`)

### CODE REVIEW

File: pycam/json2yml.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+def read_json(json_file_name: str) -> dict:
+    """
+    Opens and reads a JSON file
+    """
+    with open(json_file_name) as reader:
+        data = json.load(reader)
+        return data
+
+def format_data(data: dict) -> dict:
```
Comment (jinglingxing) [inline pycam/json2yml.py:29]: Here we might need a docstring to explain this function since the `format_data` function is long.

File: README.md [hunk 2]
```diff
@@ -11,8 +11,11 @@ Extract the archive or clone the repository.
 
 Graphical Interface: `pycam/run_gui.py`
 
-Scripted Toolpath Processing: `pycam/run_cli.py FLOW_SPECIFICATION_FILE`
+Scripted Toolpath Processing: `pycam/run_cli.py FLOW_SPECIFICATION_FILE.yml`
```
Comment (jinglingxing) [inline README.md:14]: In `README.md`, the [FAQ](http://pycam.sourceforge.net/faq.md) link has expired. The new link should be [this](http://pycam.sourceforge.net/faq/)

File: pycam/json2yml.py (hunk 1 above)
Comment (jalovisko) [reply pycam/json2yml.py:29]: This is fixed. However, I feel like I can improve other existing docstrings

File: README.md (hunk 2 above)
Comment (jalovisko) [reply README.md:14]: Nice catch! This is also done

Comment (jinglingxing) [reply README.md:14]: Great! I think I finished my review. Don't know the next step of your PR.

Comment (jalovisko) [reply README.md:14]: Thanks @jinglingxing, I'll merge this one then!

**Pull request merged**